shared/src/commonMain/kotlin/db/database/user/UserDatabase.kt

```diff
@@ -4,10 +4,10 @@ import app.cash.sqldelight.db.SqlDriver
 import db.chargehub.User
 import db.chargehub.UserDbQueries
 import db.database.GenericDatabaseOperations
-import db.networking.request.CreateUserRequest
+import db.networking.request.UserRequest
```

Comment:
I saw this change in 3 different pr's already. Make sure the branching is correct and that you merge into the branch you branched from, not always to develop